Sources/ETCD/ETCDClient.swift

```diff
@@ -120,5 +123,43 @@ public final class EtcdClient: @unchecked Sendable {
     public func put(_ key: String, value: String) async throws {
         try await put(key.utf8, value: value.utf8)
     }
+    
+    public func watch<Result>(_ key: some Sequence<UInt8>, _ operation: (WatchAsyncSequence) async throws -> Result) async throws -> Result {
+           let request = [Etcdserverpb_WatchRequest.with { $0.createRequest.key = Data(key) }]
+           let watchAsyncSequence = WatchAsyncSequence(watchClient.watch(request))
+           return try await operation(watchAsyncSequence)
+    }
+    
+    public func watch<Result>(_ key: String, _ operation: (WatchAsyncSequence) async throws -> Result) async throws -> Result {
+        let request = [Etcdserverpb_WatchRequest.with { $0.createRequest.key = Data(key.utf8) }]
```

Comment:
We can just call the other method here.
```suggestion
       try await self.watch(key.utf8, operation)
```